pkgs/by-name/an/antimatter-dimensions/package.nix

```diff
@@ -0,0 +1,71 @@
+{ buildNpmPackage
+, electron
+, fetchFromGitHub
+, lib
+, makeDesktopItem
+, writeScriptBin
+, electronAppName ? "Antimatter Dimensions"
+}:
+
+let
+ dummyElectronApp = ./app.js;
```

Comment:
Why is this needed?

Reply:
build of the project doesn't provide app.js, and that file is used to change the directory where data is stored
if you know another way to change the electron application name, like with environment variables, I am happy to use it.

Reply:
Ok, let's make that clear in the comments, and please also mention that if in future this issue is resolved upstream, this workaround is to be removed.